tools/benchmark/src/mocha/memoryTestRunner.ts

```diff
@@ -116,6 +118,19 @@ export interface MemoryTestObjectProps extends MochaExclusiveOptions, Titled, Be
 	 * Use a lower number to drop the highest/lowest measurements.
 	 */
 	samplePercentageToUse?: number;
+
+	/**
+	 * The baseline memory usage to compare against for the test, which is used to determine if the test regressed.
+	 * If not specified, the test will not be compared against a baseline and will only be run to measure the memory usage.
+	 * @remarks Should be specified in bytes.
+	 */
+	readonly baselineMemoryUsage?: number | undefined;
```

Comment:
Should be handled by the `?`.
```suggestion
	readonly baselineMemoryUsage?: number;
```